shared/constants/chat.js

```diff
@@ -438,10 +439,18 @@ function getBrokenUsers (participants: Array<string>, you: string, metaDataMap:
 }
 
 function clampAttachmentPreviewSize ({width, height}: AttachmentSize) {
-  const maxSize = Math.max(width, height)
-  return {
-    width: maxAttachmentPreviewSize * (width / maxSize),
-    height: maxAttachmentPreviewSize * (height / maxSize),
+  const ratio = width / height
+
+  if (height > width) {
+    return {
+      height: clamp(height, maxAttachmentPreviewSize),
+      width: clamp(height, maxAttachmentPreviewSize) * ratio,
+    }
+  } else {
+    return {
+      height: clamp(width, maxAttachmentPreviewSize) / ratio,
```

Comment:
I would explicitly multiply by `(height / width)` here, would be a little clearer since `ratio` is only used once either way